.github/workflows/ios-sandbox-deployment-dev.yml

```diff
@@ -0,0 +1,123 @@
+# ******************************************************************
+# This workflow will trigger once a PR goes into main from a branch
+# that does NOT have a prefix of `releases/`
+# ******************************************************************
+
+name: Deploy Dev Sandbox Apps
+
+on:
+  pull_request_target:
+    types:
+      - closed
+    branches:
+      - main
+
+  workflow_dispatch:
+  repository_dispatch:
+    types: [on-demand-testflight-dev]
+env:
+  VERSION: "2.0.0"
+
+jobs:
+  test:
+    runs-on: macos-latest
+    if: github.event.pull_request.merged && startsWith(github.head_ref, 'releases/') != true
+    steps:
+      - name: Branch Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up ruby env
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.7.6
+          bundler-cache: true
+
+      - name: Restore cached Pods
+        uses: actions/cache/restore@v3
+        with:
+          path: Pods
+          key: ${{ runner.os }}-pods-${{ hashFiles('**/Podfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-pods-
+
+      - name: CocoaPod Install
+        run: pod install
+
+      - name: Save Updated Pods
+        id: cache-primes-save
```

Comment:
id name is confusing- change primes to pods?

Reply:
yes good catch, I copied from their docs and didn't change that part 😅